blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,88 @@
+
+With my virtual machine running, I was able to use the VM as if I was on a Windows computer. From here, I could follow the ["System Requirements"](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies) and ["Getting Started"](https://microsoft.github.io/react-native-windows/docs/getting-started) documentation for React Native for Windows. Parallels had access to my local files, so I did not need to redownload any source code to get started developing. When I was finished with my development session, I was able to close the virtual machine and the machine's state was saved for next time.
+
+![](assets/2024-07-18-running-rnw-from-a-mac/Screenshot-Parallels.jpg)
+
+## Takeaways about Parallels
+
+In less than an hour, I was able to spin up a Windows VM on my Mac and use that environment to build a React Native for Windows app. Windows VMs allowed me to quickly experiment with adding Windows support to my existing React Native app because of the low level of overhead to get a VM up and running.
+
+## Creating a Windows VM using Microsoft Dev Box
+
+The second Windows VM option I tried was [Microsoft Dev Box](https://azure.microsoft.com/products/dev-box/). Microsoft Dev Box is a [Microsoft Azure](https://azure.microsoft.com) product that allows developers to spin up a Windows VM from most modern computers including MacOS devices.
+
+Microsoft Dev Box supports:
+
+1. Cloud-based virtual desktop infrastructure.
+1. Project-based dev box configurations which allow for code repositories and tools for your current task to be preinstalled.
+1. Region-specific dev boxes to help ensure that dev team members have a high-fidelity experience anywhere in the world.
+1. Secured cloud workstations via Microsoft Intune.
+
+Before you get started creating a VM with Microsoft Dev Box, there are a couple troubleshooting callouts I want to mention:
+
+1. You will need an Azure subscription that is not the ["free trial"](https://azure.microsoft.com/free/) subscription. You must upgrade to the ["Pay As You Go"](https://azure.microsoft.com/pricing/purchase-options/pay-as-you-go/) subscription to use Microsoft Dev Box, but you can use the $200 of free credits that you gained via the "free trial" subscription as initial payment within the “Pay As You Go” plan. If you try to create a Dev Box Definition with the "free trial" Azure subscription, you will receive a "Quota has been reached" error.
+1. You will also need a work or school Microsoft Account (MSA). Personal or guest MSAs cannot be used with the Dev Box portal at this time.
+1. Your machine will need to be managed with [Microsoft Intune](https://www.microsoft.com/security/business/Microsoft-Intune).
```

Comment:
The virtual machine you create or the (presumably macOS) device you are using?

Reply:
The physical machine you are using must be managed by Intune.

Reply:
Maybe then say "Your macOS device will need to be managed..."?
(ignore me if I'm being too pedantic, I'm just imaging a reaction from macOS devs about signing up their dev devices for Intune management.